FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Extensions/ConfigurationRegistrationExtensions.cs

```diff
@@ -78,7 +82,7 @@ private static void ValidateConfiguration(this IServiceCollection services)
                 .GetRequiredService<IOptions<SchemaConfiguration>>()
                 .Value;
 
-            if (schemaConfiguration.EnableCustomizedSchema && string.IsNullOrEmpty(schemaConfiguration.CustomizedSchemaImageReference))
+            if (schemaConfiguration.EnableCustomizedSchema && string.IsNullOrEmpty(schemaConfiguration.ContainerRegistry.SchemaImageReference))
```

Comment:
Maybe `IsNullOrWhitespace` is more appropriate for string type?

Reply:
Sure, updated.